src/python/grpcio/support.py

```diff
@@ -44,8 +44,10 @@
 int main(int argc, char **argv) { return 0; }
 """
 C_PYTHON_DEV_ERROR_MESSAGE = """
-Could not find <Python.h>. This could mean the following:
-  * You're on Ubuntu and haven't `apt-get install`ed `python-dev`.
+Could not find <python.h>. This could mean the following:
+  * You're on Ubuntu and haven't `apt-get install python-dev`.
```

Comment:
@soltanmm actually 'run' is proper here: http://verbix.com/webverbix/English/run.html